samples/SocketsSample/LineInvocationAdapter.cs

```diff
@@ -1,91 +0,0 @@
-// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
Is this going away completely or will be replaced? Because if is going away the corresponding javascript code should be removed as well.